src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,35 @@
+package duke.command;
+
+import duke.data.TaskList;
+import duke.data.exception.DukeException;
+import duke.data.task.Task;
+import duke.storage.Storage;
+import duke.ui.Ui;
+
+/**
+ * Deletes indexed task from TaskList.
+ */
+public class DeleteCommand extends Command {
+    private int index;
+    public DeleteCommand(String details) {
+        this.index = Integer.valueOf(details);
+    }
+    /**
+     * Executes the command given by the user.
+     *
+     * @param tasks   to be modified
+     * @param ui      to display changes
+     * @param storage to interact with as necessary
+     * @throws DukeException
```

Comment:
Should the parameter description start with caps and have a fullstop at the end? Also, should they be indented at the same level or be just right after the parameter name? Perhaps can add why the DukeException might be thrown at the back of @throws, according to the Java coding standard. I noticed this same issue in several other places too.

Reply:
The coding standard seems to state as such.

Good point pointing out about the @throws.